src/WebSockets.jl

```diff
@@ -360,18 +360,18 @@ function handle_control_frame(ws::WebSocket, wsf::WebSocketFragment)
         ws.state = CLOSED
         try
             locked_write(ws.socket, true, OPCODE_CLOSE, !ws.server, UInt8[])
-        catch
+        catch e
```

Comment:
Actually assigning the error could be expensive, and it's not actually used?
Or is the compiler smart enough to avoid doing unnecessary work here?